package.json

```diff
@@ -40,10 +45,11 @@
     "react": ">=16.8.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.13.16",
-    "@babel/preset-env": "^7.13.15",
-    "@babel/preset-react": "^7.13.13",
-    "@babel/preset-typescript": "7.13.0",
+    "@babel/cli": "^7.15.4",
```

Comment:
no babel-core?

Reply:
umm... I always thought the cli has babel core as dep already because it works without installing it. But maybe it still needs to be added. I just checked the docs and their example shows adding both. I have added it too in https://github.com/Open-Attestation/decentralized-renderer-react-components/pull/62/commits/1fc5225216ea8dc99018eaeb18c9ebce0d908c37.